samples/system-test/quickstart.test.js

```diff
@@ -62,7 +66,7 @@ describe('execute the quickstart', () => {
     assert.match(stdout, /successfully deleted/);
   });
 
-  after(async () => {
+  afterEach(async () => {
```

Comment:
This will bite us when we add more tests here?  (or maybe do `beforeEach` for bucket creation, not `before`?)